reference/roadmap.md

```diff
@@ -15,6 +15,6 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 
 | Release | Target Date | What to expect| 
 |---------|-------------|---------------|
-| 1.23.0 | August 14 2024 | * Improvements to VHAL support<br/> *Overhaul of Anbox Cloud Appliance snap<br/>  * AAOS enhancements<br/> * Support for multiple subclusters in the web dashboard<br/> * Android security updates for August 2024<br/> * Bug fixes|
 | 1.23.1 | September 18 2024 | * Android security updates for September 2024<br/> * Bug fixes |
 | 1.23.2 | October 16 2024 | * Android security updates for October 2024<br/> * Bug fixes |
+| 1.24.0 | November 13 2024 | * VHAL improvements<br/> * Support for generic application streaming<br/> * IBT integration<br/> * Android security updates for November 2024<br/> * Bug fixes
```

Comment:
Lets drop `Support for generic application streaming` and `IBT integration here`.

You can add 
* Android 14 images
* Support for additional upscaling algorithms in the streaming SDK
* Reworked charms